src/pages/index.js

```diff
@@ -183,6 +187,15 @@ const IndexPage = () => {
           </a>
           .
         </p>
+        <h3 className="font-bold mt-8" id="schedule">
+          1.2 Schedule
+        </h3>
+        <div className="mt-4">
+          <Schedule isDarkMode={isDarkMode} speakers={speakers} />
+          <div className="flex items-center justify-center text-lg mt-4">
+            <p>Figure 1: Schedule of Protocol Berg, 2025. </p>
+          </div>
+        </div>
```

Comment:
This block needs to be moved down after 1.1 Speakers

Reply:
I really think Schedule needs to be above the speakers